chihaya.go

```diff
@@ -26,16 +24,22 @@ import (
 
 	// See the README for how to import custom drivers.
 	_ "github.com/chihaya/chihaya/backend/noop"
+	"github.com/mrd0ll4r/logger"
+	"log"
 )
 
 var (
-	maxProcs   int
-	configPath string
+	maxProcs        int
+	configPath      string
+	logLevel        int
+	logFileLocation bool
 )
 
 func init() {
 	flag.IntVar(&maxProcs, "maxprocs", runtime.NumCPU(), "maximum parallel threads")
 	flag.StringVar(&configPath, "config", "", "path to the configuration file")
+	flag.IntVar(&logLevel, "level", int(logger.LevelInfo), "level of logging, where 0=Everything, 1=Trace, 2=Debug, 3=Info, 4=Warnings, 5=Fatal")
```

Comment:
Can we make this a string and have them literally pass the name of the log levels?